contracts/TwabRewards.sol

```diff
@@ -168,20 +215,21 @@ contract TwabRewards is ITwabRewards {
         uint8 _currentNumberOfEpochs = _promotion.numberOfEpochs;
 
         require(
-            _numberOfEpochs < (type(uint8).max - _currentNumberOfEpochs),
+            _numberOfEpochs <= (type(uint8).max - _currentNumberOfEpochs),
             "TwabRewards/epochs-over-limit"
         );
 
         uint8 _extendedNumberOfEpochs = _currentNumberOfEpochs + _numberOfEpochs;
         _promotions[_promotionId].numberOfEpochs = _extendedNumberOfEpochs;
 
-        uint256 _extendedAmount;
+        uint256 _amount;
 
         unchecked {
-            _extendedAmount = _numberOfEpochs * _promotion.tokensPerEpoch;
+            _amount = _numberOfEpochs * _promotion.tokensPerEpoch;
```

Comment:
Why is this unchecked?

Reply:
`_numberOfEpochs` is a uint8 and `_promotion.tokensPerEpoch` is uint256, so it can't overflow since _amount is a uint256.

Reply:
Same as above

Reply:
Done: https://github.com/pooltogether/v4-periphery/pull/48/commits/0c7a7d0876b19120f26d68f58b5b6be4a092225e